README.md

```diff
@@ -20,6 +20,16 @@ See the [API docs](http://www.rubydoc.info/gems/plek) for full details of the AP
 
 `bundle exec rake`
 
+### Environment variables
+
+The base URL Plek uses for each service can be set using environment variables.
+
+Plek will use any variables set matching this pattern:
+
+`PLEK_SERVICE_` + the service name, uppercased with any hyphens converted to underscores + `_URI`.
+
+For example, the variable for `static` would be `PLEK_SERVICE_STATIC_URI`.
+
```

Comment:
There are a few more it would be worth documenting:
```
To override the development environment base domain, set `DEV_DOMAIN`. The default is `dev.gov.uk`. The environment can be set using either `RAILS_ENV` or `RACK_ENV`.

You can prepend strings to the hostnames generated using: `PLEK_HOSTNAME_PREFIX`.

Override the asset URL with: `GOVUK_ASSET_ROOT`. The default is to generate a URL for the `static` service.

Override the website root with `GOVUK_WEBSITE_ROOT`. The default is to generate a URL for the `www` service.
```

I'm happy for you to add something or I can push another commit onto this branch if you'd prefer